dataland-backend/src/main/kotlin/org/dataland/datalandbackend/repositories/StoredCompanyRepository.kt

```diff
@@ -49,6 +51,67 @@ interface StoredCompanyRepository : JpaRepository<StoredCompanyEntity, String> {
     )
     fun searchCompanies(@Param("searchFilter") searchFilter: StoredCompanySearchFilter): List<StoredCompanyEntity>
 
+    /**
+     * A function for querying companies by search string:
+     * - searchString: If not empty, only companies that contain the search string in their name are returned
+     * (Prefix-Matches are ordered before Center-Matches,
+     * e.g. when searching for "a" Allianz will come before Deutsche Bank)
+     */
+    @Query(
+        nativeQuery = true,
+        value =
+        "WITH filtered_text_results as (" +
+            // Fuzzy-Search Company Name
+            "(SELECT company_id, company_name AS company_name," +
```

Comment:
The `AS company_name` seems unnecessary

Reply:
I think so too. CI should fail if something is wrong here. So I will push the change.